public/CSS/style.sss.example

```diff
@@ -0,0 +1,84 @@
+.App {
```

Comment:
Good shout using an example stylesheet, however, should not probably be included in commits/git - maybe consider moving it into .gitignore